asapdiscovery/docking/docking.py

```diff
@@ -212,19 +244,23 @@ def run_docking_oe(
         poser.AddReceptor(du)
 
         ## Run posing
-        pose_res = oedocking.OESinglePoseResult()
-        ret_code = poser.Dock(pose_res, dock_lig)
+        pose_res = oedocking.OEPositResults()
+        ret_code = poser.Dock(pose_res, dock_lig, num_poses)
 
     ## Check results
-    if ret_code == oedocking.OEDockingReturnCode_Success:
-        if dock_sys == "posit":
-            posed_mol = pose_res.GetPose()
-            posit_prob = pose_res.GetProbability()
-
-        ## Get the Chemgauss4 score (adapted from kinoml)
-        pose_scorer = oedocking.OEScore(oedocking.OEScoreType_Chemgauss4)
-        pose_scorer.Initialize(du)
-        chemgauss_score = pose_scorer.ScoreLigand(posed_mol)
+    if (
+        ret_code == oedocking.OEDockingReturnCode_Success
+        and dock_sys == "posit"
+    ):
+        posed_mols = []
+        posit_probs = []
+        posit_methods = []
+        for r in pose_res.GetSinglePoseResults():
```

Comment:
best to use a more verbose variable name than `r` for readability

Reply:
good point, fixed in [0ac9c57](https://github.com/choderalab/covid-moonshot-ml/pull/93/commits/0ac9c57fd745f515a5333d27673081821e5bc707)